back/src/controllers/categories/categories.go

```diff
@@ -38,5 +39,5 @@ func sendCategories(ctx *fiber.Ctx, categories []model.Category) error {
 func SelectCategoryByID(category *model.Category, ctx *fiber.Ctx) error {
 	categoryID := ctx.Params("categoryID")
 
-	return config.Database.First(&category, categoryID).Error
+	return config.Database.Preload("Products").First(&category, categoryID).Error
```

Comment:
This also could be used to make a global helper to SelectModelByID, what do you think ?

Reply:
You can make a helper just for database actions

Reply:
As discussed, this could be later on refactored in another MR